### ISSUE TITLE
Outputs huge number of boxes with almost same confidence after training

### ISSUE THREAD

**Issue by aelnouby** [thtrieu/darkflow#398]
I am trying to fine-tune tiny-yolo for only the "Person" class. After training what always happens is that it detects huge number of boxes, with very close confidences all in the range 0.5-0.6.

I tried to simplify the problem by over-fitting to just 4 examples and I get loss around 0.2 before finishing, however the same issue persists.

The last portion of the cfg file:

> [convolutional]
size=1
stride=1
pad=1
filters=30
activation=linear

>[region]
anchors = 1.08,1.19,  3.42,4.41,  6.63,11.38,  9.42,5.11,  16.62,10.52
bias_match=1
classes=1
coords=4
num=5
softmax=1
jitter=.2
rescore=1

>object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .5
random=1

![2007_000170](https://user-images.githubusercontent.com/8495451/30338601-2ab8e7cc-97ba-11e7-8194-89a494a74080.jpg)
![2007_000027](https://user-images.githubusercontent.com/8495451/30338599-29e64d8a-97ba-11e7-96a4-6f36e09b5cc7.jpg)

**Comment by deamonDevs** [comment]
I am also facing the same issue...
